system/Log/Logger.php

```diff
@@ -98,10 +98,8 @@ class Logger implements LoggerInterface
 
     /**
      * Caches logging calls for debugbar.
-     *
-     * @var array
      */
-    public $logCache;
+    public array $logCache;
```

Comment:
This is a breaking change for class extensions. Please explain or revert.

Reply:
Hi @MGatner , thanks for your reply.
I have now restored this attribute to its original state.
The reason I will make this change is because of the following.

`$ignoreErrors[] = [
	'message' => '#^Property CodeIgniter\\\\Log\\\\Logger\\:\\:\\$logCache \\(array\\) on left side of \\?\\? is not nullable\\.$#',
	'count' => 1,
	'path' => __DIR__ . '/system/Debug/Toolbar/Collectors/Logs.php',
];
`

Reply:
That is not a "type specified" error. We can tackle that in separate PR.